### PR TITLE
Refine README.md

### PR DESCRIPTION
**Pull request by hzy46**
Fix https://github.com/microsoft/pai/issues/4492

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,17 +13,28 @@ OpenPAI is an open source platform that provides complete AI model training and
 
 **OpenPAI [v0.14.0](./RELEASE_NOTE.md#july-2019-version-0140) has been released!**
 
+A brief comparison between `v0.14.0` and `v1.0.0`:
+
+|                   | `v0.14.0`               | `v1.0.0`               |
+| ----------------- | ----------------------- | ---------------------- |
+| Architecture      | Kubernetes + Yarn       | Kubernetes             |
```
Comment (abuccts) [inline README.md:20]: Yarn -> Hadoop YARN

File: README.md [hunk 2]
```diff
@@ -13,17 +13,28 @@ OpenPAI is an open source platform that provides complete AI model training and
...
+| Scheduler         | Yarn Scheduler          | HiveD / K8S default    |
```
Comment (abuccts) [inline README.md:21]: same

File: README.md (hunk 1 above)
Comment (hzy46) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (hzy46) [reply README.md]: Done

**Pull request merged**